common/src/main/cpp/extensions/openxr_fb_scene_extension_wrapper.cpp

```diff
@@ -98,7 +98,7 @@ void OpenXRFbSceneExtensionWrapper::_on_instance_destroyed() {
 }
 
 const PackedStringArray &OpenXRFbSceneExtensionWrapper::get_supported_semantic_labels() {
-	static PackedStringArray semantic_labels = String(SUPPORTED_SEMANTIC_LABELS).split(",");
+	static PackedStringArray semantic_labels = String(SUPPORTED_SEMANTIC_LABELS).to_lower().split(",");
```

Comment:
Can you add a comment describing why we're converting to lower case (e.g: to match Godot's automatically converting them to lower-case when saving properties).

Reply:
Sure, I added a comment at both the places where we convert the semantic labels to lower-case